app/src/androidTest/java/org/kiwix/kiwixmobile/help/HelpFragmentTest.kt

```diff
@@ -17,27 +17,38 @@
  */
 package org.kiwix.kiwixmobile.help
 
+import android.os.Build
 import androidx.test.internal.runner.junit4.statement.UiThreadStatement.runOnUiThread
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.Before
 import org.junit.Test
 import org.kiwix.kiwixmobile.BaseActivityTest
 import org.kiwix.kiwixmobile.R
 
 class HelpFragmentTest : BaseActivityTest() {
 
+  @Before
+  fun waitForIdle() {
+    UiDevice.getInstance(InstrumentationRegistry.getInstrumentation()).waitForIdle()
+  }
+
   @Test
   fun verifyHelpActivity() {
-    runOnUiThread { activityRule.activity.navigate(R.id.helpFragment) }
-    help {
-      clickOnWhatDoesKiwixDo()
-      assertWhatDoesKiwixDoIsExpanded()
-      clickOnWhatDoesKiwixDo()
-      clickOnWhereIsContent()
-      assertWhereIsContentIsExpanded()
-      clickOnWhereIsContent()
-      clickOnHowToUpdateContent()
-      assertHowToUpdateContentIsExpanded()
-      clickOnHowToUpdateContent()
-      clickOnSendFeedback()
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
+      runOnUiThread { activityRule.activity.navigate(R.id.helpFragment) }
```

Comment:
OK, but what comes after `TIRAMISU`? How do we test this? I don't understand the point of this condition.

Reply:
hi @gouri-panda ,

as i mentioned above also i have by pass some test which are failing on android 13 due to unavailability of ```default apis```. I have created https://github.com/kiwix/kiwix-android/issues/3085 for this also , so once its available we can filter out this conditions.

Reply:
hi @gouri-panda , @kelson42 ,

i have upgraded ```androidx:test:core``` from ```androidx.test:core:1.4.0``` to ```androidx.test:core:1.5.0-alpha02``` because in android 13 it's giving ```ActivityNotFoundException``` exception with ```ActivityScenario``` and ```ActivityScenarioRule``` you can find more information https://github.com/android/android-test/issues/1412 here.

Reply:
What are default apis and google apis exactly? Do you have some kind of official documentation i can read?

I hardly can understand how android default api can vanish, so @MohitMaliFtechiz you should be more precise about what exactly has vanished.

On the other side I don't want Kiwix to rely on propriatery online google api, so here again I wonder why and for what we do so.

Reply:
@MohitMaliDeveloper @kelson42 I have only doubt in this regard. Otherwise, it is good to merge.

Reply:
I'll approve the PR since it's urgency

Reply:
hi @kelson42  @gouri-panda ,googleapis and default are system images , default one does not contain service like google map etc, ```Support for Google specific Android APIs like the Android Google maps API, are not provided with the standard Android system images ```
https://stackoverflow.com/questions/23416448/difference-between-google-apis-x86-system-image-and-google-apis-arm-system-im